src/config/base.rs

```diff
@@ -139,3 +140,40 @@ impl Base {
         }
     }
 }
+
+static DEFAULT_TEST_REGION: &str = "127.0.0.1:19000";
+
+fn parse_region(maybe_region: Option<String>) -> s3::Region {
+    match maybe_region {
+        Some(region) if region.contains("://") => {
+            let (_proto, host) = region.split_once("://").unwrap();
+            s3::Region::Host(host.to_string())
+        }
+        Some(region) if region.is_empty() => s3::Region::Default,
+        Some(region) => s3::Region::Region(region),
+        // This needs to match the region configuration that was used to generate the cassettes in
+        // `src/tests/http-data`.
+        None => s3::Region::Host(DEFAULT_TEST_REGION.to_string()),
```

Comment:
it seems a bit unexpected to me that "empty string" and "no string" behave entirely differently 🤔

Reply:
Yeah, that's probably too confusing on second thought. I'm going to change it so an empty string is handled identically to a missing one, and anyone who wants to use the "default" region can just say `us-east-1` explicitly.